### DISCUSSION TITLE
Sensitivity of change detection tiggers unwanted drafts

### DISCUSSION THREAD

**Discussion by lenvanessen** [craftcms/cms#10024] [Ideas]
Many of our editors use chrome extensions like Grammarly and some others as well. This means that the moment they open an article, too proofread for instance, Grammarly will touch the input field and Craft will create a new provisionary draft when actually no changes have been made. 

Not sure how to avoid this, maybe we can implement a different way of detecting changes here.

**Comment by brandonkelly** [comment]
I just installed Grammarly to test, and am unable to reproduce. Even if there are suggested changes, Grammarly does not appear to be adding or modifying any form inputs, which is what Craft checks when determining whether to create/autosave a draft.


<img width="1110" alt="An Edit Entry page showing that Grammarly has opinions, but Craft has not created a provisional draft for the entry." src="https://user-images.githubusercontent.com/47792/139068025-837bb16d-a0b9-46f1-bdad-38e0abf741c0.png">

Once I accept a suggestion, a provisional draft is created, but I wouldn’t consider that to be unexpected. The content has changed, whether I made the change directly, or instructed Grammarly to do it.

**Comment by lenvanessen** [comment]
@brandonkelly I'll test a bit further when this happens, I'll let you know if I have a better set of steps to reproduce
